src/czml3/widget.py

```diff
@@ -77,21 +81,24 @@
 class CZMLWidget:
 
     document = attr.ib(default=Document([Preamble()]))
-    
+
     cesium_version = attr.ib(default="1.79.1")
-    
+
     ion_token = attr.ib(default="no_token")
-    
+
     terrain = attr.ib(default=TERRAIN["Cesium"])
-    
+
     imagery = attr.ib(default=IMAGERY["Bing_Aerial"])
-    
+
     _container_id = attr.ib(factory=uuid4)
 
     @ion_token.validator
     def _check_ion_token(self, attribute, value):
         if value == "no_token" or value == "" or value is None:
-            warnings.warn("No ion_token Provided. Some features may not work. Please get your free token at https://cesium.com/ion/tokens", FutureWarning)
+            warnings.warn(
```

Comment:
Instead of completely halting the CZML Widget, I have just raised warnings if he does not specify ion_token. The CZML Widget displays just fine but the earth remains invisible.